src/data/actions/authentication.test.js

```diff
@@ -2,17 +2,29 @@ import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 import axios from 'axios';
 import MockAdapter from 'axios-mock-adapter';
+import Cookies from 'universal-cookie';
+
+import { login, logout } from './authentication';
 
-import login from './loginForm';
 import {
   FETCH_LOGIN_FAILURE,
   FETCH_LOGIN_REQUEST,
   FETCH_LOGIN_SUCCESS,
-} from '../constants/loginForm';
+  LOGOUT,
+} from '../constants/authentication';
 
 const mockStore = configureMockStore([thunk]);
 const axiosMock = new MockAdapter(axios);
 
+jest.genMockFromModule('universal-cookie');
+jest.mock('universal-cookie');
+
+const mockCookies = {
+  set: jest.fn(),
+  remove: jest.fn(),
+};
+Cookies.mockImplementation(() => mockCookies);
```

Comment:
This sets the Cookies constructor with our mock, so we can spy on set/remove below